docset/winserver2012-ps/adcsdeployment/install-adcscertificationauthority.md

```diff
@@ -45,73 +45,53 @@ Install-AdcsCertificationAuthority [-AllowAdministratorInteraction] [-ValidityPe
 ```
 
 ## DESCRIPTION
-The Install-AdcsCertificationAuthority cmdlet performs installation and configuration of the AD CS CA role service.
-To remove the certification authority role service use the Uninstall-AdcsCertificationAuthority cmdlet.
+The **Install-AdcsCertificationAuthority** cmdlet performs installation and configuration of the Active Directory Certificate Services (AD CS) Certification Authority (CA) role service.
+To remove the certification authority role service use the **Uninstall-AdcsCertificationAuthority** cmdlet.
 
 You can import the cmdlet by running the following commands from Windows PowerShell:
-`Import-Module ServerManager`
-`Add-WindowsFeature Adcs-Cert-Authority`
 
-To include the Certification Authority and Certificate Templates consoles in a CA installation, you must add `-IncludeManagementTools` to the end of the `AddWindowsFeature Adcs-Cert-Authority` command.
+- `Install-WindowsFeature Adcs-Cert-Authority`
 
-Int is equivalent to Int32 in the .NET Frameworkhttp://msdn.microsoft.com/en-us/library/ya5y69ds.aspx (http://msdn.microsoft.com/en-us/library/ya5y69ds.aspx).
+To include the Certification Authority and Certificate Templates consoles in a CA installation, you must use the *IncludeManagementTools* parameter at the end of the `Install-WindowsFeature Adcs-Cert-Authority` command.
+
+Int is equivalent to Int32 in the [.NET Framework](https://msdn.microsoft.com/en-us/library/ya5y69ds.aspx).
```

Comment:
Please update the URL to https://docs.microsoft.com/dotnet/csharp/language-reference/builtin-types/built-in-types